packages/docs/community/release-process/smart-contracts.md

```diff
@@ -72,11 +72,23 @@ yarn verify-deployed -n $NETWORK -b $PREVIOUS_RELEASE -f
 
 ### Check Backward Compatibility
 
-Use the following script to compile candidate and current release contracts to check storage layout and ABI backwards compatibility, subject to the following exceptions:
+This script performs some automatic checks to ensure that smart contract
+versions have been updated correctly.
+
+Specifically, it compiles candidate and current release contracts to check storage layout and ABI backwards compatibility, subject to the following exceptions:
```

Comment:
Wdyt of:
```
Specifically, it compiles the latest and candidate releases and compares smart contract:

1. Storage layout, to detect storage version changes
2. ABI, to detect major and minor version changes
3. Bytecode, to detect patch version changes

Finally, it checks release candidate smart contract version numbers and requires that they have been updated appropriately since the latest release.

The script generates a detailed report on version changes in JSON format.
```